content/docs/Configuration/_index.md

```diff
@@ -36,16 +38,17 @@ future version (>0.64.0) of Packit. We recommend projects to explicitly set
 `specfile_path` or rely on the `<downstream_package_name>.spec` default, by
 setting `downstream_package_name`.
 
-#### ~~upstream_project_name~~ (deprecated)
-
-(*string*) Deprecated since packit-0.7.0, use [`upstream_package_name`](#upstream_package_name) instead.
-
 #### upstream_package_name
 
 (*string*) Name of the upstream repository (e.g. in PyPI), defaults to the
 name of the GitHub repository; it's used when working with the upstream project,
 as a release archive name and as a directory name in that archive.
 
+{{< hint info >}}
+Previously known as `upstream_project_name` and has been deprecated since Packit
```

Comment:
If I didn't know I'd probably think for a millisecond about which one has been deprecated.

```suggestion
Previously known as `upstream_project_name`, which has been deprecated since Packit
```